Gopkg.toml

```diff
@@ -97,7 +97,7 @@ required = [
 
 [[constraint]]
   name = "github.com/pmorie/go-open-service-broker-client"
-  version = "=0.0.11"
+  version = "0.0.12"
```

Comment:
This changes the semantics from "require exactly v0.0.11" to "require v0.0.12 up to but not including v1.0.0". I recommend switching to `~0.0.12` which I'm guessing is what you intended.

Reply:
I wish I could say that was an accident on my part.  I must admit I wasn't aware of that and actually thought the equal was an error as none of the others were expressed this way.  Thanks very much for catching and correcting me on this - - now I know (& read more of the dep doc)!  I've reverted back to the =